src/Graphics/OpenGLContext/GLSL/glsl_CombinerProgramBuilderCommon.h

```diff
@@ -0,0 +1,133 @@
+		void _writeFragmentHeaderCalcLight(std::stringstream& ssShader) const override;
+		void _writeFragmentMain2Cycle(std::stringstream& ssShader) const override;
+		void _writeFragmentMain(std::stringstream& ssShader) const override;
+		void _writeFragmentBlendMux(std::stringstream& ssShader) const override;
+		void _writeShaderCoverage(std::stringstream& ssShader) const override;
+		void _writeFragmentCorrectTexCoords(std::stringstream& ssShader) const override;
+		void _writeFragmentReadTexMipmap(std::stringstream& ssShader) const override;
+		void _writeFragmentCallN64Depth(std::stringstream& ssShader) const override;
+		void _writeFragmentRenderTarget(std::stringstream& ssShader) const override;
+		void _writeShaderFragmentMainEnd(std::stringstream& ssShader) const override;
+		void _writeShaderCalcLight(std::stringstream& ssShader) const override;
+		void _writeShaderNoise(std::stringstream& ssShader) const override;
+		void _writeShaderDither(std::stringstream& ssShader) const override;
+		void _writeShaderWriteDepth(std::stringstream& ssShader) const override;
+		void _writeShaderN64DepthCompare(std::stringstream& ssShader) const override;
+		void _writeShaderN64DepthRender(std::stringstream& ssShader) const override;
+
+		typedef std::unique_ptr<ShaderPart> ShaderPartPtr;
+		ShaderPartPtr m_blender1;
+		ShaderPartPtr m_blender2;
+		ShaderPartPtr m_blenderAlpha;
+		ShaderPartPtr m_legacyBlender;
+		ShaderPartPtr m_clamp;
+		ShaderPartPtr m_signExtendColorC;
+		ShaderPartPtr m_signExtendAlphaC;
+		ShaderPartPtr m_signExtendColorABD;
+		ShaderPartPtr m_signExtendAlphaABD;
+		ShaderPartPtr m_alphaTest;
+		ShaderPartPtr m_callDither;
+
+		ShaderPartPtr m_vertexHeader;
+		ShaderPartPtr m_vertexEnd;
+		ShaderPartPtr m_vertexRect;
+		ShaderPartPtr m_vertexTexturedRect;
+		ShaderPartPtr m_vertexTriangle;
+
+		ShaderPartPtr m_fragmentHeader;
+		ShaderPartPtr m_fragmentGlobalVariablesNotex;
+		ShaderPartPtr m_fragmentHeaderNoise;
+		ShaderPartPtr m_fragmentHeaderWriteDepth;
+		ShaderPartPtr m_fragmentHeaderCalcLight;
+		ShaderPartPtr m_fragmentHeaderMipMap;
+		ShaderPartPtr m_fragmentHeaderDither;
+		ShaderPartPtr m_fragmentHeaderDepthCompare;
+		ShaderPartPtr m_fragmentMain;
+		ShaderPartPtr m_fragmentMain2Cycle;
+		ShaderPartPtr m_fragmentBlendMux;
+		ShaderPartPtr m_fragmentCorrectTexCoords;
+		ShaderPartPtr m_fragmentReadTexMipmap;
+		ShaderPartPtr m_fragmentCallN64Depth;
+		ShaderPartPtr m_fragmentRenderTarget;
+		ShaderPartPtr m_shaderFragmentMainEnd;
+
+		ShaderPartPtr m_shaderNoise;
+		ShaderPartPtr m_shaderDither;
+		ShaderPartPtr m_shaderWriteDepth;
+		ShaderPartPtr m_shaderCalcLight;
+		ShaderPartPtr m_shaderN64DepthCompare;
+		ShaderPartPtr m_shaderN64DepthRender;
+		ShaderPartPtr m_shaderCoverage;
+
+		u32 m_combinerOptionsBits;
+
+		GLuint m_vertexShaderRect;
+		GLuint m_vertexShaderTriangle;
+		GLuint m_vertexShaderTexturedRect;
+		GLuint m_vertexShaderTexturedTriangle;
+	};
```

Comment:
Indent is unnecessary here. Please remove left indent for the code inside `namespace `brackets, as it is done in cpp file.